scripts/publish.cjs

```diff
@@ -10,49 +10,51 @@ function bumpMinorVersion(version) {
 const currentVersion = packageJson.version;
```

Comment:
Maybe I wasn't clear but the idea is that we remove this file and move publishing into the nx.json config.

Reply:
I understand that now. Will work on doing that now!